app/models/episode.rb

```diff
@@ -42,7 +42,7 @@ def self.by_prx_story(story)
   end
 
   def publish_updated
-    podcast.publish_updated if podcast
+    podcast.publish_updated if podcast && published?
```

Comment:
I agree with @ryan, also the episode shouldn't be aware of what the podcast will do with this message, it should just poke it